pkg/visibility/server.go

```diff
@@ -77,7 +78,7 @@ func applyVisibilityServerOptions(config *genericapiserver.RecommendedConfig) er
 	o.SecureServing.BindPort = 8082
 	// The directory where TLS certs will be created
 	o.SecureServing.ServerCert.CertDirectory = "/tmp"
-
+	o.Admission.DisablePlugins = []string{validatingadmissionpolicy.PluginName}
```

Comment:
Ideally the visibility server can be minimal and any other plugins which are not required can be disabled. I believe the only plugin we would need would be NamespaceLifecycle, ServiceAccount, ResourceQuota.

Reply:
That is interesting. What are your motivations for trying to get rid of only the VAP plugin?
What if when we enable only `NamespaceLifecycle`, `ServiceAccount`, and `RersourceQuota`?

Additionally, why do we need `ResourceQuota` plugin? IIUC, the visibility server doesn't touch `ResourceQuta` everywhere.

Reply:
> What are your motivations for trying to get rid of only the VAP plugin?

There are 2 benefits to this:
1. Latest releases of Kueue would become compatible with previous versions of K8s - (like 1.29) where VAP is not enabled by default in the admission plugin.
2. Currently Kueue has CRUD perms for VAP API which is not required by the controller at all. So this would reduce unnecessary privileges being given to the controller. 

> Additionally, why do we need ResourceQuota plugin? IIUC, the visibility server doesn't touch ResourceQuta everywhere.
Fair. Ideally it would be good to spawn the visibility server with just what's required rather than enabling all the plugins by default. If `ResourceQuota` isn't required we can remove it from the list too.

Reply:
Nice find!  Makes sense to me to enable just the features we are using.

Reply:
> Fair. Ideally it would be good to spawn the visibility server with just what's required rather than enabling all the plugins by default. If ResourceQuota isn't required we can remove it from the list too.

That makes sense. Could you open an issue for that?